backend/models/cans.py

```diff
@@ -386,7 +385,9 @@ class ServicesComponent(BaseModel):
 
     __tablename__ = "services_component"
 
-    id = Column(Integer, Identity(), primary_key=True)
+    # start Identity at 4 to allow for the records load with IDs
+    # in agreements_and_blin_data.json5
+    id = Column(Integer, Identity(start=4), primary_key=True)
```

Comment:
Would be better not to do this - since in PROD this will create a gap (starting at 4) - but it's minor

Reply:
I agree.  I'll leave it for now but follow up later with some alternatives.